src/ipc/ipc4/helper.c

```diff
@@ -669,6 +669,7 @@ int ipc4_create_mng_for_chain(struct ipc4_chain_dma *cdma)
 	struct comp_dev *dev;
 
 	drv = ipc4_get_drv((uint8_t *)&uuid);
+
 	if (!drv)
 		return IPC4_INVALID_RESOURCE_ID;
 
```

Comment:
please drop unrelated white-space changes. In fact in my subjective perception most of the SOF code doesn't have an empty line before checking return code like
```
ret = fn();
if (ret < 0)
```